backend/de.metas.business/src/main/java/de/metas/costrevaluation/CostRevaluationDocumentHandler.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.costrevaluation;
+
+import de.metas.document.engine.DocumentHandler;
+import de.metas.document.engine.DocumentTableFields;
+import de.metas.document.engine.IDocument;
+import de.metas.util.Services;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.model.I_M_CostRevaluation;
+import org.compiere.model.I_M_CostRevaluationLine;
+import org.compiere.util.TimeUtil;
+
+import java.io.File;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.List;
+
+public class CostRevaluationDocumentHandler implements DocumentHandler
+{
+
+	private static I_M_CostRevaluation extractForecast(final DocumentTableFields docFields)
+	{
+		return InterfaceWrapperHelper.create(docFields, I_M_CostRevaluation.class);
+	}
+
+	private final ICostRevaluationDAO costRevaluationDAO = Services.get(ICostRevaluationDAO.class);
```

Comment:
Please, declare the services at the beginning of the class, before adding the methods.